homework-submission/myapp/src/components/itemList.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Item from "./item";
+import { observer, inject } from "mobx-react";
+
+@inject("todoStore")
+@observer
+class ItemList extends React.Component {
+  render() {
+    const { todoStore } = this.props;
+
+    const createItems = todoStore.Data.map((item, index) => {
```

Comment:
`id` is for identifying used to be, while index for what it means, don't we want to identify ? if yes 😃 then we may use `item.id` instead of `id` as a key?